journal/modernized/32-epistles.adoc

```diff
@@ -218,17 +225,21 @@ or encumbered with the things of this life; but be ready for sufferings,
 which may attend that nation before many be aware, that we may all be ready,
 as Christ`'s freemen, to drink that cup which the Lord is pleased to put into our hands,
 for the trial of our faith, which is more precious than gold.
+
+[.signed-section-closing]
```

Comment:
I prefer to keep closings very short, if possible.  Sort of like "yours truly" or "sincerely".  It's a fuzzy line, but I in general will not pull something out as a closing unless it's pretty short and a little more close to one of those modern examples in it's content.

For this one, let's modify it and format it to match the standalone (not Friends Library) edition, as shown here, which means pulling most of this sentence back up, adding a comma, and just using "Your friend and brother", as shown here:

![2019-04-02_12-07-08](https://user-images.githubusercontent.com/7050938/55418234-395a5500-5540-11e9-8b67-e67155b05fc6.png)